### PR TITLE
hb-04 task done

### PR DESCRIPTION
**Pull request by less08**

### CODE REVIEW

**Review by AleksandrPok — APPROVED** [review]

File: src/main/java/com/dev/cinema/dao/MovieSessionDao.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+import com.dev.cinema.model.MovieSession;
+import java.time.LocalDate;
+import java.util.List;
+
+public interface MovieSessionDao {
+    MovieSession add(MovieSession movieSession);
+
+    List<MovieSession> findAvailableSessions(Long movieId, LocalDate date);
+
```
Comment (KseniiaMakarova) [inline src/main/java/com/dev/cinema/dao/MovieSessionDao.java]: redundant empty line

File: src/main/java/com/dev/cinema/model/CinemaHall.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
+package com.dev.cinema.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class CinemaHall {
```
Comment (KseniiaMakarova) [inline src/main/java/com/dev/cinema/model/CinemaHall.java:11]: add `@Table` annotation here to have the `proper_name` for the table

File: src/main/java/com/dev/cinema/model/MovieSession.java [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+import java.time.LocalDateTime;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+
+@Entity
+public class MovieSession {
```
Comment (KseniiaMakarova) [inline src/main/java/com/dev/cinema/model/MovieSession.java:15]: same here

File: src/main/java/com/dev/cinema/model/MovieSession.java [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne(fetch = FetchType.LAZY)
+    private Movie movie;
+    @ManyToOne(fetch = FetchType.LAZY)
+    private CinemaHall cinemaHall;
+    private LocalDateTime showTime;
```
Comment (KseniiaMakarova) [inline src/main/java/com/dev/cinema/model/MovieSession.java:24]: let's use `@JoinColumn` annotation to have `proper_column_names` for columns that consist of several words

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**